webgme.js

```diff
@@ -46,8 +47,9 @@ function addToRequireJsPaths(gmeConfig) {
                 try {
                     items = fs.readdirSync(basepaths[j]);
                     if (items.indexOf(componentNames[i]) !== -1) {
-                        componentPaths[componentName + '/' + componentNames[i]] = path.relative(requireJsBase,
-                            path.resolve(basepaths[j]));
+                        componentPath = path.relative(requireJsBase, path.resolve(basepaths[j]))
+                            .replace(/^client\/js/, 'js');
+                        componentPaths[componentName + '/' + componentNames[i]] = componentPath;

```

Comment:
Do we really need the replace part here? It can potentially mess up custom paths.